src/FakeClient/FakeClientMain.cpp

```diff
@@ -263,7 +287,19 @@ int main(int argc, char* argv[]) {
   absl::SetProgramUsageMessage("Orbit fake client for testing");
   absl::ParseCommandLine(argc, argv);
 
+  LOG("Starting Client");
+  uint32_t duration_s = absl::GetFlag(FLAGS_duration);
+  FAIL_IF(duration_s == 0, "Specified a zero-length duration");
+  FAIL_IF((absl::GetFlag(FLAGS_instrument_path).empty()) !=
+              (absl::GetFlag(FLAGS_instrument_offset) == 0),
+          "Binary path and offset of the function to instrument need to be specified together");
+
   uint32_t process_id = absl::GetFlag(FLAGS_pid);
+  if (process_id == 0) {
+    FAIL_IF(absl::GetFlag(FLAGS_pid_file_path).empty(), "A PID or a path to a file is needed.");
```

Comment:
I would declare a variable to hold the result of `absl::GetFlag(FLAGS_pid_file_path)`.

Reply:
done